lib/mapUtils.js

```diff
@@ -66,3 +66,12 @@ export const getProductData = function async(store) {
     }
   });
 };
+
+// Necessary to take into account different screen sizes
+// 32px: Width of left and right margins
+// 12px: Extra space for padding between items
+// 130px: Size of 'See Products' button
+// 48px: Size of EBTBar
+export const getMaxWidth = function sync(screenWidth, ebt, seeProducts) {
+  return screenWidth - 32 - 12 + (ebt ? -48 : 0) + (seeProducts ? -130 : 0);
```

Comment:
We should add a @todo to see how we can minimize hard-coded values here, and throughout the app (if any exists now)! No good solution as of yet - might just require specific/clear documentation during handoff.